src/lib/rest/HttpStatusCode.h

```diff
@@ -36,6 +36,7 @@ typedef enum HttpStatusCode
 {
   SccNone                   = 0,     // Undefined
   SccOk                     = 200,   // Success
+  // FIXME P10: SccFound 302 will be removed at the end once #787 gets implemented
```

Comment:
302 will be removed when you implement the new serviceRoutines part for postQueryContext and updateQueryContext. By the moment, we need to keep to avoid awful compliation errors.